packages/cli/src/platform/auth/show.ts

```diff
@@ -41,9 +45,36 @@ export class Show implements Command {
         `,
       },
     })
+
+    let sensitiveData: { token: string | null } = {
+      token: null,
+    }
```

Comment:
```suggestion
    let sensitiveData: { token: string | undefined } = {
      token: undefined,
    }
```

Reply:
Coincidentally I had this initially, but then `messages.resouce()` ends up logging `undefined` to the console. In contrast, it filters out nulls, thus the `null`: https://github.com/prisma/prisma/blob/0e92f9d40caf8888ca8bb04fbe893b48689168b9/packages/cli/src/platform/_lib/messages.ts#L35

I'd instead simply just add the property if the condition is met to avoid it needing to filter out nulls in the first place, but then I'd also need to add the relevant logic for the second param of `message.resource()` (e.g. `shouldShowSensitive ? { token } : {}`).

Overall just seemed simpler to go for null.

Reply:
You can also expand that condition to filter out both `null` and `undefined`. I personally don't have a strong preference and the code is fine with me as it currently is too.